### ISSUE TITLE
Individual channel?

### ISSUE THREAD

**Issue by rwols** [renatoGarcia/gdb-imshow#2]
I have an image with 4 channels and I'd like to inspect each channel separately. Is this possible?

**Comment by renatoGarcia** [comment]
Currently there are no support to inspect channels separately. But that is a cool idea, it would be nice to show only the first channel with a command like:
`cv_imshow image 1`

For now, a workaround would be to split the image channels with C++ code, with a function like [cv::split]. But that is far from ideal, since you would have to change the code before using the debugger.

[cv::split]: https://docs.opencv.org/3.4.0/d2/de8/group__core__array.html#ga8027f9deee1e42716be8039e5863fbd9

**Comment by rwols** [comment]
Yes, I thought of the same syntax as you. I played around with the struct.unpack calls but couldn’t get it to work right away. I frequently work with CMYK images and it’d be useful to see the separate channels.
